sites/main-site/src/content/events/2025/hackathon-march-2025/bogota-colombia.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Hackathon - March 2025 (Bogota, Colombia)'
+subtitle: 'Local node of the nf-core hackathon at University of the Andes, Colombia'
+shortTitle: 'Bogota, Colombia'
+type: 'hackathon'
+startDate: '2025-03-17'
+startTime: '09:00+00:00'
+endDate: '2025-03-19'
+endTime: '17:00+00:00'
+locations:
+  - name: Bogota, Colombia
+    address: |
+      University of the Andes
+    links:
+      - https://www.loka.com/
+    geoCoordinates: [4.6533816, -74.0836333]
```

Comment:
Falls in the middle of a roundabout :sweat_smile: 
```suggestion
    geoCoordinates: [4.601261, -74.066064]
```

Uni of andes is here at least

https://maps.app.goo.gl/eVgXAzkTbGFBbAZf7